dependent-mapping/README.md

```diff
@@ -0,0 +1,30 @@
+---  
+layout: pattern  
+title: Dependent Mapping
+folder: dependent-mapping
+permalink: /patterns/dependent-mapping/  
+categories: Behavioral
+tags:
+- Database
+---  
+
+## Intent
+The basic idea behind Dependent Mapping is that one class (the dependent) relies upon some other class (the owner) for its database persistence. Each dependent can have only one owner and must have one owner.
+
+A dependent may itself be the owner of another dependent. In this case the owner of the first dependent is also responsible for the persistence of the second dependent. You can have a whole hierarchy of dependents controlled by a single primary owner.
+
+Writing and saving of dependents is left to the owner, and there are no outside references, updates to the dependents can be handled through deletion and insertion. Thus, if you want to update the collection of dependents you can safely delete all rows that link to the owner and then reinsert all the dependents.
```

Comment:
Please provide the explanation section as well. See the other patterns how they should be formatted.